playwright/fabric-custom.test.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, test } from '@playwright/test';
+
+const viewport = { width: 1600, height: 1000 };
+
+const widths = ['360', '740', '980', '1300', '100%'];
+
+test.describe('Fabric Custom visual regression testing', () => {
+	test('Get reference screenshots', async ({ page }) => {
+		await page.setViewportSize(viewport);
+
+		await page.goto(
+			'https://guardian.github.io/commercial-templates/ssr/fabric-custom/',
+			{
+				waitUntil: 'networkidle',
+			},
+		);
+
+		for (const breakpoint of widths) {
+			const referenceTemplateLocator = page
+				.frameLocator(`[name='width-${breakpoint}']`)
+				.locator('#creative');
+			// check that the template is present on the page
+			await referenceTemplateLocator.isVisible();
+			// scroll to it
+			await referenceTemplateLocator.scrollIntoViewIfNeeded();
+			// take a reference screenshot
+			await referenceTemplateLocator.screenshot({
+				animations: 'disabled',
+				path: `./playwright/reference-images/Fabric-custom-${breakpoint.replace('%', '')}.png`,
+			});
+		}
+	});
+
+	test('Compare PR templates to reference screenshots', async ({ page }) => {
+		await page.setViewportSize(viewport);
+
+		await page.goto('http://localhost:7777/ssr/fabric-custom', {
```

Comment:
Minor - is it worth creating a shared const for the port number

Reply:
Yeah good shout! I'll do the same for the templates site as well